src/main/java/com/snowflake/kafka/connector/internal/SnowflakeSinkServiceV1.java

```diff
@@ -108,7 +108,7 @@ public void startTask(final String tableName, final TopicPartition topicPartitio
     String stageName = Utils.stageName(conn.getConnectorName(), tableName);
     String nameIndex = getNameIndex(topicPartition.topic(), topicPartition.partition());
     if (pipes.containsKey(nameIndex)) {
-      LOGGER.error("task is already registered, name: {}", nameIndex);
+      LOGGER.warn("task is already registered, name: {}", nameIndex);
```

Comment:
this log line actually doesnt make sense, `task is already registered with {} partition` makes more sense.

Reply:
Not familiar with this code path, but seems like the expectation is that this shouldn't happen and that's why it was logged as an error?

Reply:
that's true, it shouldnt happen. will only happen if close didnt wipe off map. so keeping it error makes sense.

Reply:
This was opened as an issue.

Since having a task already registered is not a breaking scenario, i think it does make sense that it should be a warning instead of an error?